### PR TITLE
Explain Apsystems entities in tables rather than lists

### PR DESCRIPTION
**Pull request by mawoka-myblock**
## Proposed change

- Move all of the data/actions provided by the apsystems integration into datatables (main part)
- Change wording to support all inverters even with different outputs

This PR includes a rewrite of #34083 so I has to be the `next` branch

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/123356
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes https://github.com/home-assistant/home-assistant.io/pull/33944#discussion_r1692635777, closes https://github.com/home-assistant/home-assistant.io/issues/33972

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Enhanced clarity and detail in the APsystems integration documentation.
  - Introduced new sections categorizing Numerical and Binary Sensors with detailed tables for easier reference.
  - Simplified the inverter output limit description for better understanding.
  - Added a new Settings section outlining inverter configuration options.

### CODE REVIEW

**Comment by mawoka-myblock** [comment]
Hope you don't mind getting into this conversation @c0ffeeca7

**Comment by c0ffeeca7** [comment]
> Hope you don't mind getting into this conversation @c0ffeeca7

Love it!
One comment: Do you think this could work as a list, instead of a table? I realize it is very common to use tables for this. But they don't look that great on mobile:
image

Maybe something like this.
- total_power (W): Total current output of the inverter
- lifetime_production_p (kWh): Lifetime production of first input

**Comment by mawoka-myblock** [comment]
Tbh, that's not really my problem as the other tables from other doc pages look just as bad on mobile.

**Comment by mawoka-myblock** [comment]
To be more precise: Your criticism is fair, but as said above, I don't consider this to be my or your problem. This is more like an issue as IMO, those tables should overflow and then should be scrollable horizontally.

File: source/_integrations/apsystems.markdown [hunk 1]
```diff
@@ -18,20 +18,39 @@ ha_codeowners:
   - '@SonnenladenGmbH'
 ---
 
-The **APsystems** {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter. It also allows you to set the output limit to any number between 30 and 800 watts.
-The following data is provided by the integration:
+The **APsystems** {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter. It also allows you to set the output limit to anything above 30 watts.
+
+## Sensors
+
+### Numerical Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/apsystems.markdown]:
```suggestion
### Numerical sensors

```

File: source/_integrations/apsystems.markdown [hunk 2]
```diff
@@ -18,20 +18,39 @@ ha_codeowners:
...
+| lifetime_production_p1 | kWh | Lifetime production of first input
+| lifetime_production_p2 | kWh | Lifetime production of second input
+| lifetime_production | kWh | Lifetime production of both inputs combined
+| total_power_p1 | W | Current input on first input
+| total_power_p2 | W | Current input on second input
+| today_production | kWh | Today's production of both inputs combined
+| today_production_p1 | kWh | Today's production of first input
+| today_production_p2 | kWh | Today's production of second input
+
+### Binary Sensors
```
Comment (c0ffeeca7) [inline source/_integrations/apsystems.markdown]:
```suggestion
### Binary sensors

```

File: source/_integrations/apsystems.markdown [hunk 3]
```diff
@@ -18,20 +18,39 @@ ha_codeowners:
...
+| Sensor ID  | Description
+|---|---|
+| off_grid_status | On when the inverter is not connected to the power grid
+| dc_1_short_circuit_error_status | Short circuit detected on first input
+| dc_2_short_circuit_error_status | Short circuit detected on second input
+| output_fault_status | Output because of any error deactivated
+
+## Settings
```
Comment (c0ffeeca7) [inline source/_integrations/apsystems.markdown:48]:
```suggestion
## Settings

```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mawoka-myblock :+1:

**Comment by mawoka-myblock** [comment]
Thanks for fixing these small things and the approving review!

**Pull request merged**